src/main/java/service/FileWriterService.java

```diff
@@ -0,0 +1,17 @@
+package service;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class FileWriterService {
+    public void writeToFile(List<String> lines, String filePath) {
+        try (FileWriter writer = new FileWriter(filePath)) {
+            for (String line : lines) {
+                writer.write(line + "\n");
```

Comment:
Please use System.lineSeparator() instead of system dependant hardcoded symbol
```suggestion
                writer.write(line + System.lineSeparator());
```